### DISCUSSION TITLE
【Internal Discussion】新生代 Web3 Buidler 究竟需要什么？

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#74] [Workshop]
- 新生代 Web3 开发者的需求到底有哪些？

> 痛点：
> 
> 痛苦的点，当用户在使用产品或服务的时候抱怨、不满的、让人感到痛苦的接触点（touchpiont）。
> 
> 痒点：
>
> 有了很好，没了问题也不大。
>
> 爽点：
>
> 高效的即时满足，在特定的场景下，因欲求不满会引发某些行动，满足了就很爽，然后强化认知，追求更爽。
>
> 参考文章：http://www.woshipm.com/pd/4234595.html

- 将新生代开发者作为用户，进行因素分析

![image](https://user-images.githubusercontent.com/12784118/164909404-72f2b967-e9e7-4de6-b247-8607051ea375.png)

第一类：保健因素。

保健因素的特征，是如果做得不好，人们会很不满意；但哪怕做得再好，最多人们会觉得还行吧，不会特别满意。

他举了餐厅的例子，说比如卫生，往往就是保健因素。

如果餐厅很脏，我们会非常不满。但卫生搞得再好，你也不会觉得，这是一家顶尖的餐厅。

第二类：线性因素

线性因素的特征，是如果做得不好，人们会很不满意；如果好了，满意度可能就很高。

例如餐厅的上菜速度，对很多人可能就是个线性因素。上菜太慢，会很恼火；上菜很快，会感觉不错。

第三类：激励因素

激励因素的特征，是即使没有这个，人们也不会不满。但如果有了，满意度就大为提升。

例如今天我们去餐厅，老板突然说你们老客户，送一份大龙虾，那可能就成了激励因素，觉得这餐厅挺好。

因为超出了自己的预期。

当然，如果每次老板都送龙虾，那估计久而久之就变成一个保健因素了。有了没感觉，没有觉得不爽。

参考文章：https://hi.daozhuixuetang.com/xiu-lian-yi-ling-zhan-kai-jie-jue-wen-ti-shi-dui-hua.html

**Comment by gofenix** [comment]
报名
我感觉主要就是好用的开发者工具，现在的工具都有些没有注重开发者体验。
减少样板代码，增加local开发的体验，同时也方便部署

**Comment by qiwihui** [comment]
🙋‍♂️

**Comment by 99Kies** [comment]
🙋‍♂️

**Comment by keeei1218** [comment]
🙋

**Comment by leeduckgo** [comment]
Raw Text 出炉啦~有需求的来自取哦😁：

> https://r8jmm3f9xe.feishu.cn/docs/doccnjHSQB6YRwOumE2F9wSIzDh
